CONTRIBUTING.md

```diff
@@ -59,33 +43,83 @@ pnpm i
 pnpm run setup
 ```
 
-Note for Windows:
-Use the latest version of [Git Bash](https://gitforwindows.org/)
+> 💡 For Windows users: use the latest version of [Git Bash](https://gitforwindows.org/).
 
 ## Building packages when you make changes
 
 In the root directory:
 
 - `pnpm run setup` will install and build all the packages
 - `pnpm -r run build` (-r for recursive) will build all the packages
-- `pnpm -r run dev` (-r for recursive) will build all the packages without running `tsc` (Fastest)
+- `pnpm -r run dev` (-r for recursive) will build all the packages, without running `tsc`
+- `pnpm run watch` will continuously build any packages that have been modified, without running `tsc` (Fastest)
 
 In a package directory, like `packages/client`:
 
 - `pnpm run build` will build the package
 - `pnpm run dev` will build the package without running `tsc` (Fastest)
 
-Note: Our builder is ESbuild
+> 💡 Our builder is built on top of esbuild
 
 ## Prisma Client
 
 ### First contribution
 
-1. `cd packages/client`
-1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
-1. `cd fixtures/blog`
-1. `npx prisma db push --skip-generate` will create the database structure
-1. `ts-node main`
+Create a reproduction folder for developing, trying a new feature, or a fix.
+
+#### Setting up a locally-linked development folder
+
+Set up a local project that will be linked to the local packages.
+
+```sh
+cd reproductions && pnpm install
+# Copy a template from the reproduction folder
+cp -r basic-sqlite my-repro && cd my-repro
+# Ensure that the db and the schema are synced
+pnpx prisma db push --skip-generate
+# Do some code changes, compile, then try it out
+pnpx prisma generate && pnpx ts-node index.ts
+```
+
+> 💡 This works best when compiling with `pnpm run watch` in the background.
+
+> 💡 In any successful setup `pnpx prisma -v` should return version `0.0.0`.
+
+<details>
+  <summary><b>Alternatives</b></summary>
+  
+  #### Detailed steps for a locally-linked dev folder
+  ```sh
+  cd reproductions
+  mkdir my-repro
+  cd my-repro
+  pnpm init -y
+  pnpm add ../../prisma/packages/client
+  pnpm add -D ../../prisma/packages/cli
```

Comment:
When executing this from the root folder of the git repo it will result in these lines in the package.json
```
  "dependencies": {
    "client": "link:../../prisma/packages/client"
  },
  "devDependencies": {
...
    "cli": "link:../../prisma/packages/cli",
```

And here it seems that `link:` is extra and causes `pnpx` to ask to install latest